hs/CourseographyFacebook.hs

```diff
@@ -44,25 +52,29 @@ credentials :: FB.Credentials
 credentials = FB.Credentials (T.pack courseographyUrl) appId appSecret
 
 perms :: [FB.Permission]
-perms = []
+perms = ["publish_actions"]
+
+postPhoto :: FB.UserAccessToken -> FB.UserId -> IO Response
+postPhoto (FB.UserAccessToken _ b _) id_ = withSocketsDo $ withManager $ \m -> do
+    fbpost <- parseUrl $ "https://graph.facebook.com/v2.2/me/photos?access_token=" ++ T.unpack b
```

Comment:
`T.++` does not exist. Why would we use a Text here? `parseUrl` takes in a String.